### ISSUE TITLE
Ability to log in by entering your email address instead of your username

### ISSUE THREAD

**Issue by bcj** [mouse-reeve/bookwyrm#531]
**Is your feature request related to a problem? Please describe.**
Many websites allow you to log-in with your username or your email address. I tend to prefer email address because it's consistent and I don't always remember what username I was able to get for a particular site. Even though the bookwyrm login clearly says `username` I often enter my email address without thinking.

**Describe the solution you'd like**
I would like to be able to enter my username _or_ my email address to identify my account when logging in.

**Describe alternatives you've considered**
* Taking two attempts to log in every time
* Just get used to entering my username on the login page
* create a new account where my account name is the same as my email address

Basically, I realize this is me not following instructions and it isn't a big deal but it would be nice
